letsencrypt/translations/en.yaml

```diff
@@ -37,5 +37,11 @@ configuration:
   dns:
     name: DNS
     description: DNS Provider configuration
+  key_type:
+    name: Key Type
+    description: Select the certificate key type.
```

Comment:
This is obsolete now.

```suggestion
```

Reply:
This should still be valid (other than fixing name).

Reply:
Keeping this for the help text in the config page.